composer.json

```diff
@@ -1,7 +1,8 @@
 {
     "name": "keymanapi/keyman.com",
     "require": {
-        "sentry/sdk": "^2.1"
+        "sentry/sdk": "^2.1",
+        "erusev/parsedown-extra": "^0.8.1"
```

Comment:
Should we add parsedown as well, not just the sub-dependency?

Reply:
afaict, parsedown-extra will automatically install parsedown

Reply:
yeah, I expect so, but I'd like to list it as a depedency all the same so we have deterministic versions.